README.md

```diff
@@ -1 +1,329 @@
-# flight-log-be
+# Flight Log - REST API
+![Workflow Badge](https://github.com/Flight-Log/flight-log-be/actions/workflows/run-tests.yml/badge.svg)
+
+ 
+## About Flight Log
+Flight Log is a mobile application that allows those in the aviation industry to easily track their flight hours. Users will love the simple and easy-to-use app that will enable them to log their flights digitally instead of in a physical notebook. 
```

Comment:
```suggestion
Flight Log is a mobile application that allows aviation industry professionals to easily track their flight hours and history. Users will love the simple and easy-to-use app that enables them to log their flights digitally instead of in a physical notebook. 
```

Reply:
```suggestion
Flight Log is a mobile application that allows those in the aviation industry to easily track their flight hours. Users will love the simple and easy-to-use app that will enable them to log their flights digitally instead of in a physical notebook. 

This RESTful API back end is built with Python and Django, and utilizes the [Django-REST-Framework](https://www.django-rest-framework.org/) toolkit.
```